cmd/daemon/main.go

```diff
@@ -47,6 +47,9 @@ var (
 	enablePprof      = flag.Bool("enable-pprof", true, "Enable pprof for daemon.")
 	pluginConfigFile = flag.String("plugin-config-file", "/kruise/CredentialProviderPlugin.yaml", "The path of plugin config file.")
 	pluginBinDir     = flag.String("plugin-bin-dir", "/kruise/plugins", "The path of directory of plugin binaries.")
+
+	//todo: After the feature is stable, the default value should also be restricted, e.g. 5.
```

Comment:
todo ->  TODO

plz comment in the code how to set the flag, what is the recommended worker number